### PR TITLE
added missing type annotations

### PR DESCRIPTION
**Pull request by aashish2054**
Added missing type annotation to make code more readable and easy to understand.

### CODE REVIEW

File: zio-kafka-test/src/test/scala/zio/kafka/ProducerSpec.scala [hunk 1]
```diff
@@ -4,19 +4,23 @@ import org.apache.kafka.clients.producer.ProducerRecord
...
 import zio.test._
 
 object ProducerSpec extends ZIOKafkaSpec {
   override val kafkaPrefix: String = "producerspec"
 
-  def withConsumerInt(subscription: Subscription, settings: ConsumerSettings) =
+  def withConsumerInt(
+    subscription: Subscription,
+    settings: ConsumerSettings
+  ): ZIO[Any with Scope, Throwable, Dequeue[Take[Throwable, CommittableRecord[String, RuntimeFlags]]]] =
```
Comment (svroonland) [inline zio-kafka-test/src/test/scala/zio/kafka/ProducerSpec.scala:23]: This should be `CommittableRecord[String, Int]`.

File: zio-kafka/src/main/scala/zio/kafka/consumer/ConsumerSettings.scala [hunk 2]
```diff
@@ -16,7 +16,7 @@ case class ConsumerSettings(
   rebalanceListener: RebalanceListener = RebalanceListener.noop,
   restartStreamOnRebalancing: Boolean = false
 ) {
-  private[this] def autoOffsetResetConfig = offsetRetrieval match {
+  private[this] def autoOffsetResetConfig: Map[_ <: String, String] = offsetRetrieval match {
```
Comment (svroonland) [inline zio-kafka/src/main/scala/zio/kafka/consumer/ConsumerSettings.scala:19]: Let's make this `Map[String, String]`

**Review by svroonland — CHANGES_REQUESTED** [review]
Thanks, a few minor comments

**Comment by aashish2054** [comment]
all checks passed, can this be merged?

File: zio-kafka/src/main/scala/zio/kafka/consumer/ConsumerSettings.scala (hunk 2 above)
Comment (aashish2054) [reply zio-kafka/src/main/scala/zio/kafka/consumer/ConsumerSettings.scala]: done

File: zio-kafka-test/src/test/scala/zio/kafka/ProducerSpec.scala (hunk 1 above)
Comment (aashish2054) [reply zio-kafka-test/src/test/scala/zio/kafka/ProducerSpec.scala]: done

**Review by svroonland — APPROVED** [review]

**Pull request merged**